File: lib/layer2_core/dark/e_secondary_container_dark.dart
import '../../layer1_raw/design_token.dart';
import '../../layer1_raw/dark/system.dart';
import '../../layer1_raw/dark/standard.dart';
import '../../layer1_raw/dark/blue.dart';
import '../../layer1_raw/dark/green.dart';
import '../../layer1_raw/dark/red.dart';
import '../../layer1_raw/dark/nothing.dart';

class ESecondaryContainerDark {
  static DesignToken getToken(dynamic activeTheme) {
    switch (activeTheme.toString().split('.').last) {
      case 'system': return csyd27;
      case 'standard': return cstd27;
      case 'blue': return cbd27;
      case 'green': return cgd27;
      case 'red': return crd27;
      case 'nothing': return cnod27;
      default: return cstd27;
    }
  }
}

File: android/src/main/kotlin/com/example/design_engine/layer2_core/dark/ESecondaryContainerDark.kt
package com.example.design_engine.layer2_core.dark
import com.example.design_engine.R
import com.example.design_engine.layer3_logic.AppTheme
object ESecondaryContainerDark {
    fun getResId(activeTheme: AppTheme): Int {
        return when (activeTheme) {
            AppTheme.SYSTEM -> R.string.csyd27
            AppTheme.STANDARD -> R.color.cstd27
            AppTheme.BLUE -> R.color.cbd27
            AppTheme.GREEN -> R.color.cgd27
            AppTheme.RED -> R.color.crd27
            AppTheme.NOTHING -> R.color.cnod27
        }
    }
}
